packages/web-components/src/components/ic-side-navigation/ic-side-navigation.tsx

```diff
@@ -424,7 +424,7 @@ export class SideNavigation {
 
     const navStyles = {
       gap: window.getComputedStyle(navItemLink).gap,
-      iconWidth: window.getComputedStyle(navItemSVG).width,
+      iconWidth: navItemSVG ? window.getComputedStyle(navItemSVG).width : "0",
```

Comment:
Maybe add an optional chaining to the method object and add it to the rest of the methods that call the same function? 

Best would be to add the check to the function itself. This way you guarantee any calls to the function will return something anywhere

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining